java/src/jmri/jmrit/vsdecoder/VSDecoderPreferences.java

```diff
@@ -87,8 +86,8 @@ public void load(org.jdom2.Element e) {
         if ((a = e.getAttribute("isAutoStartingEngine")) != null) {
             setAutoStartEngine(a.getValue().compareTo("true") == 0);
         }
-        if ((a = e.getAttribute("isAutoLoadingDefaultVSDFile")) != null) {
-            setAutoLoadDefaultVSDFile(a.getValue().compareTo("true") == 0);
+        if ((a = e.getAttribute("isAutoLoadingVSDFile")) != null) {
```

Comment:
Is this a setting that's common among users? If so, it might be good to read the old attribute if the new one isn't present.

When I do these kind of changes in LogixNG, I add a temporary code to read the old attribute if the new one isn't present, and I add a comment to that code with the JMRI version that has the change. Later, if I want to, I can remove that temporary code when enough time has passed.

Another solution to this problem is to add a note about it in the release notes, that the user should check this setting.

Reply:
Great suggestions, Daniel, thanks!